CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+ * [#826](https://github.com/toptal/chewy/pull/826): Fixed mismatch argument names at update_mapping rake task ([@AgeevAndrew][])
```

Comment:
Let's mention the issue:
```suggestion
 * [#825](https://github.com/toptal/chewy/issues/825): Fix mismatch argument names at update_mapping rake task ([@AgeevAndrew][])
```
And please also add your GH link to the list at the very end, in the alphabetical order
```
[@AgeevAndrew]: https://github.com/AgeevAndrew
```

Reply:
Ok. Done 👌